packages/react-app/src/hooks/index.ts

```diff
@@ -0,0 +1,11 @@
+import {useDeviceWidthCheck} from "./useDeviceWidthCheck"
```

Comment:
lets use `export * from './useXYZ'`

in that way we need not keep track of exports.